web/component/adsSticky/view.jsx

```diff
@@ -42,9 +42,12 @@ export default function AdsSticky(props: Props) {
   const shouldLoadSticky = shouldShowAds && !gScript && !inIFrame();
 
   function shouldShowAdsForPath(pathname, isContentClaim, isChannelClaim, authenticated) {
+    const pathIsSensitive = ['reset', 'resetpassword', 'verify', 'walletpassword'].some((path) =>
+      pathname.includes(path)
+    );
     // $FlowIssue: mixed type
     const pathIsCategory = Object.values(homepageData).some((x) => pathname.startsWith(`/$/${x?.name}`));
-    return pathIsCategory || isChannelClaim || isContentClaim || pathname === '/';
+    return !pathIsSensitive && (pathIsCategory || isChannelClaim || isContentClaim || pathname === '/');
   }
```

Comment:
Looks good

A few suggestions, but not necessarily needed:

:small_blue_diamond: Use the `PAGES` constants so that it's easier to find usages in the future, and we'll only need to change the constant side if the pathname changes.

:small_blue_diamond: Use "startsWith(`/$/${PAGES.xxx}`)" instead of `includes` to be exact, otherwise paths like `rewards/verify` and other redirect stuff would be captured as well (although not a big deal in this case, since it's just not showing ads).

Reply:
Thanks, updated.